src/components/GradeItemAccordion.tsx

```diff
@@ -8,45 +8,55 @@ export interface GradeItemAccordionInterface {
 	updateItem: (item: CourseItem) => void;
 }
 
-const Accordion = styled.tr`
-	background-color: #ffffff;
+const Accordion = styled.div`
 	color: #4f4f4f;
-	padding: 2em;
-	justify-content: center;
-	line-height: 8em;
-	transition: line-height 14em ease-out;
-	border: #ffffff;
-	text-decoration: none;
-	border: none;
+
+	display: grid;
+	grid-template-columns:
+		minmax(3em, 1fr) minmax(6em, 5fr) minmax(6em, 5fr)
+		minmax(6em, 5fr) minmax(6em, 5fr);
+	grid-template-rows: 6em;
+	grid-template-areas: 'arrow item weight grade duedate';
 `;
-const RowComponent = styled.td`
-	border: #ffffff;
+const RowComponent = styled.div`
 	font-weight: bold;
 	font-size: large;
 	border: none;
-	margin: 1.5em;
+
+	background-color: ${({ theme }) => theme.colors.main[0]};
 `;
 const DropDownArrow = styled.button`
 	border: none;
 	outline: none;
-	background-color: #ffffff;
 	font-size: 2em;
+	background-color: ${({ theme }) => theme.colors.main[0]};
 `;
-const AccordionExtended = styled.tr`
+const AccordionExtended = styled.div`
 	font-weight: bold;
 	font-size: large;
 	color: #4961e1;
 	border: none;
+	background-color: ${({ theme }) => theme.colors.main[0]};
+
+	display: grid;
+	grid-template-columns:
+		minmax(3em, 1fr) minmax(6em, 5fr) minmax(6em, 5fr)
+		minmax(6em, 5fr) minmax(6em, 5fr);
+	grid-template-rows: 4em;
+	grid-template-areas: 'arrow item weight grade duedate';
 `;
-const AccordionExtendedComponent = styled.td`
-	background-color: #ffffff;
+
+const AccordionExtendedComponent = styled.div`
 	border: none;
+	padding-right: 1.5em;
 `;
 
 const DeleteButton = styled(DelButton)`
-	background-color: #ffffff;
 	color: #b80f0a;
-	margin-right: 2em;
+	width: auto;
+	font-weight: bold;
+	padding-left: 0em;
+	padding-right: 0em;
```

Comment:
Hack, and I think standard practice, you can just put `0` instead of `0em`, `0px` and the like.